src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs

```diff
@@ -10,6 +10,7 @@ public enum Architecture
         Arm,
         Arm64,
         Wasm,
-        S390x
+        S390x,
+        LoongArch64
```

Comment:
Could you please also add this one to `src/libraries/System.Runtime/ref/System.Runtime.cs`? It will fix one of the build breaks that you are seeing.

Reply:
OK, thanks